Nos/Views/Components/NoteTextEditor.swift

```diff
@@ -3,18 +3,26 @@ import Logger
 
 /// A view similar to `TextEditor` for composing Nostr notes. Supports autocomplete of mentions.
 struct NoteTextEditor: View {
-    
+
     /// A controller for the entered text.
     @Binding private var controller: NoteEditorController
-    
+
     /// The smallest size of EditableNoteText
     var minHeight: CGFloat
-    
-    init(controller: Binding<NoteEditorController>, minHeight: CGFloat) {
+
+    /// The authors who are referenced in a note in addition to those who replied to the note, if any.
+    var relatedAuthors: [Author]?
+
+    init(
+        controller: Binding<NoteEditorController>,
+        minHeight: CGFloat,
+        relatedAuthors: [Author]? = nil
```

Comment:
I usually prefer just an array that's empty, rather than an optional array. Does the optional add any value here?

```suggestion
        relatedAuthors: [Author] = []
```

I realize this requires at least a change to the relatedAuthors property.

Reply:
It being an optional doesn't change anything. I will update it.

Reply:
Thanks!